smrf/data/mysql_data.py

```diff
@@ -124,7 +124,7 @@ def get_data(self, table, station_ids, start_date, end_date, variables):
 
         self._logger.debug('Determined data time step to be %f minutes' % m)
 
-        t = date_range(start_date, end_date, timedelta(minutes=m))
+        # t = date_range(start_date, end_date, timedelta(minutes=m))
```

Comment:
Want to delete this line then all together?

Reply:
Decided to keep it and use it.